namespaces/init.go

```diff
@@ -95,7 +103,11 @@ func Init(container *libcontainer.Config, uncleanRootfs, consolePath string, pip
 
 	label.Init()
 
-	if err := mount.InitializeMountNamespace(rootfs,
+	if (cloneFlags & syscall.CLONE_NEWNS) == 0 {
```

Comment:
Okay :) it deserves comment in code then
I'm also okay with else for `NEWNET` for symmetry.